test/framework/flux.go

```diff
@@ -577,6 +592,44 @@ func (e *ClusterE2ETest) validateWorkerNodeMachineSpec(ctx context.Context, clus
 			e.T.Logf("Worker MachineTemplate values have matched expected values")
 			return nil
 		})
+	case v1alpha1.CloudStackDatacenterKind:
```

Comment:
can we pls remove this? we don't need it anymore right?

Reply:
The new flux test for cloudstack still uses it:
TestCloudStackUpgradeMulticlusterWorkloadClusterWithFlux -> runWorkloadClusterFlowWithGitOps -> UpgradeWithGitOps -> validateWorkerNodeUpdates -> validateWorkerNodeMachineSpec